buf.yaml

```diff
@@ -1,15 +1,21 @@
 ---
-version: "v1"
+version: "v2"
```

Comment:
I'm curious, did you use `buf config migrate`? https://buf.build/docs/migration-guides/migrate-v2-config-files

Reply:
Yes